examples/quasiparticle/aws/cross_cloud/cross_cloud_storage.py

```diff
@@ -0,0 +1,55 @@
+from pcf.particle.aws.s3.s3_bucket import S3Bucket
+from pcf.core import State
+from pcf.quasiparticle.cross_cloud.cross_cloud_storage import CrossCloudStorage
```

Comment:
i had to do `from pcf.quasiparticle.cross_cloud.cross_cloud_storage.cross_cloud_storage import CrossCloudStorage
` make the import available from the init file to remove the second cross_cloud_storage